guides/source/configuring.md

```diff
@@ -3335,6 +3335,8 @@ Now the behavior is clear, that we are only using the connection information in
 
 Rails comes with built-in support for [SQLite3](https://www.sqlite.org), which is a lightweight serverless database application. While Rails better configures SQLite for production workloads, a busy production environment may overload SQLite. Rails defaults to using an SQLite database when creating a new project, but you can always change it later.
 
+NOTE: Rails uses an SQLite3 database for data storage by default because it is a zero configuration database that just works. Rails also supports MySQL (including MariaDB) and PostgreSQL "out of the box", and has plugins for many database systems. If you are using a database in a production environment Rails most likely has an adapter for it.
```

Comment:
I feel like this last sentence is an invitation for "but my database doesn't appear to be supported!". It's kind of confusing because Rails only supports 3 adapters, there are other adapters in the ecosystem but they implement Rails API, not the other way around, if that makes sense.

```suggestion
NOTE: Rails uses an SQLite3 database for data storage by default because it is a zero configuration database that just works. Rails also supports MySQL (including MariaDB) and PostgreSQL "out of the box", and has plugins for many database systems.```

Reply:
For whatever it's worth, that line was introduced in b9e52636 in 2010 (and not by this PR, which simply moved this note). I'll clean it up.

Reply:
Oops I missed that. I think it's better without it anyway. Thanks!